### ISSUE TITLE
Lua memory leak: webview widgets are never collected by GC 

### ISSUE THREAD

**Issue by aidanholm** [aidanholm/luakit#238]
Something is holding onto a reference to webview widgets; references to webviews collected in a weak table will never be released, and attempts to access the webview widgets will result in `Lua error: widget 0xwhatever (webview) has been destroyed`.

**Comment by aidanholm** [comment]
Caused by reference cycle in `lib/lousy/widget/tab.lua`:
```
view -> property::uri signal handler -> local tl -> data[tl] -> view
```
Would normally be freed by `tab:destroy()`, which should be called, but isn't.

**Comment by aidanholm** [comment]
Fixed at eb6361d0
